### ISSUE TITLE
Neovim doesn't compile

### ISSUE THREAD

**Issue by ProkopRandacek** [kiss-community/community#575]
```
make[5]: Leaving directory '/root/.cache/kiss/proc/1622311/build/neovim/.deps/build/src/luajit/src'
==== Successfully built LuaJIT 2.1.0-beta3 ====
==== Installing LuaJIT 2.1.0-beta3 to /root/.cache/kiss/proc/1622311/build/neovim/.deps/usr ====
==== Successfully installed LuaJIT 2.1.0-beta3 to /root/.cache/kiss/proc/1622311/build/neovim/.deps/usr ====

Note: the development releases deliberately do NOT install a symlink for luajit
You can do this now by running this command (with sudo):

  ln -sf luajit-2.1.0-beta3 /root/.cache/kiss/proc/1622311/pkg/neovim/root/.cache/kiss/proc/1622311/build/neovim/.deps/usr/bin/luajit

make[4]: Leaving directory '/root/.cache/kiss/proc/1622311/build/neovim/.deps/build/src/luajit'
[ 79%] Completed 'luajit'
CMake Error: failed to create symbolic link '/root/.cache/kiss/proc/1622311/build/neovim/.deps/usr/bin/luajit': No such file or directory
make[3]: *** [CMakeFiles/luajit.dir/build.make:136: luajit] Error 1
```
Probably somewhere missing `mkdir -p` call.

**Comment by dilyn-corner** [comment]
I don't use `neovim`, and it's honestly too much of a headache for me to constantly be dealing with. 

It's very much adoptable right now, if you (or anyone else) would like to pick it up! I'll try and work on this, but something to keep in mind.

**Comment by AndreiSva** [comment]
Reproduced. `mkdir`ing the directory before calling make still doesn't result in a successful build :(

**Comment by dilyn-corner** [comment]
Should be fixed with 1d756260b9f5e650067584128094f88a97fdf94b -- let me know!
